selenium_testcase/testcases/utils.py

```diff
@@ -45,6 +42,9 @@ def wrapped(self, *args, **kwargs):
 
         while True:
             try:
+                # render the log file containing a screen shot on entry
```

Comment:
This was a subtle bug so the commit message could use a bit more explanation.

I can reword that if you are busy, of course.